src/sentry/lang/native/symbolicator.py

```diff
@@ -489,6 +491,7 @@ def _request(self, method, path, **kwargs):
         # required for load balancing
         kwargs.setdefault("headers", {})["x-sentry-project-id"] = self.project_id
         kwargs.setdefault("headers", {})["x-sentry-event-id"] = self.event_id
+        kwargs.setdefault("headers", {})["x-sentry-worker-pid"] = str(WORKER_PID)
```

Comment:
Since we're running in Docker, it might be the case that `getpid()` will be returning more or less the same low values (~1-30, depending on Celery concurrency) in different workers (containers). Wdyt about adding some randomness (or e.g. hostname) here?

Reply:
How about `uuid.uuid4().int` (or `.hex`)? https://docs.python.org/3/library/uuid.html?#uuid.uuid4

Reply:
Changed to a uuid4().